datafusion/sql/src/planner.rs

```diff
@@ -2386,8 +2386,14 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
             ));
         }
 
-        let query = if variable.to_lowercase() == "all" {
+        let variable_lower = variable.to_lowercase();
+        println!("{}", variable_lower);
```

Comment:
Perhaps is this a left over debugging `println!`?

Reply:
my fault, let me push the fix

Reply:
@alamb  fixed